idseq_dag/util/command.py

```diff
@@ -276,3 +275,12 @@ def scp(key_path, remote_username, instance_ip, remote_path, local_path):
         ip=instance_ip,
         remote_path=remote_path,
         local_path=local_path)
+
+def remote(base_command, key_path, remote_username, instance_ip):
```

Comment:
we already have execute_remote above this :)

Reply:
ok.